_posts/0000-01-02-pjosh0523.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
```

Comment:
“Give a man a fire and he's warm for a day, but set fire to him and he's warm for the rest of his life.”-Terry Pratchett